### PR TITLE
people/aasolim: Add Anas’ personal page

### PR DESCRIPTION
**Pull request by dcallega**

### CODE REVIEW

File: bib/aalsolim-other.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{panteater2014example,
```
Comment (igorburago) [inline bib/aalsolim-other.bib]: The only BibTeX entries in this file are the exemplary ones that
should not have remained here in the first place. If there are no
publications to be included in this file, keep it empty or delete
it altogether.

File: bib/aalsolim.bib [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@inproceedings{panteater2017bandwidth,
```
Comment (igorburago) [inline bib/aalsolim.bib]: The only BibTeX entries in this file are the exemplary ones that
should not have remained here in the first place. If there are no
publications to be included in this file, keep it empty or delete
it altogether.

File: content/people/aalsolim.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+name: Anas Alsoliman
+role: student-PhD
```
Comment (igorburago) [inline content/people/aalsolim.md]: The role id should be in lowercase.

File: content/people/aalsolim.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+status: Ph.D. Candidate
+yearjoined: 2017
+portrait: aalsolim.jpg
+badge: aalsolim-badge.jpg
+interests:
+  - Vehicular Networks
+  - Connected-Cars/Autonomous Vehicles networking applications
```
Comment (igorburago) [inline content/people/aalsolim.md:10]: Is the dash in “Connected-Cars” necessary? If this is indeed how the
name of that field of research is written, ignore this comment.

File: content/people/aalsolim.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  - Infrastructure support of smart cities and Intelligent Transportation Systems (ITS)
+education:
+  - degree: M.S. in Network Engineering and Security
+    institution: DePaul University.
```
Comment (igorburago) [inline content/people/aalsolim.md]: Please add a location: “DePaul University, Illinois, U.S.“

**Review by igorburago — CHANGES_REQUESTED** [review]

File: content/people/aalsolim.md (hunk 4 above)
Comment (dcallega) [reply content/people/aalsolim.md:10]: I did a quick research but I don't know as well. 
I'd keep it for now since he cared enough to write it.

Comment (igorburago) [reply content/people/aalsolim.md:10]: All right. Just wanted to make sure it is not a mistake, as it looks
somewhat unusual.

File: bib/aalsolim-other.bib (hunk 1 above)
Comment (igorburago) [reply bib/aalsolim-other.bib]: @dcallega: Please delete the exemplary entry from this file, as well.

**Review by igorburago — CHANGES_REQUESTED** [review]

File: content/people/aalsolim.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+name: Anas Alsoliman
+role: student-phd
+status: Ph.D. Candidate
```
Comment (igorburago) [inline content/people/aalsolim.md]: @dcallega Is that correct? Has Anas gone through candidacy?

**Review by igorburago — APPROVED** [review]
LGTM

**Pull request merged**